otredis.go

```diff
@@ -2,34 +2,67 @@ package otredis
 
 import (
 	"context"
+	"strings"
 
 	"github.com/go-redis/redis"
-	opentracing "github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go"
 	"github.com/opentracing/opentracing-go/ext"
 )
 
-// WrapRedisClient adds opentracing measurements for commands and returns cloned client
```

Comment:
could you please return documentation for this function.
All public functions must be documented.

Reply:
No prob